src/OutputWriters/fetch_output.jl

```diff
@@ -9,8 +10,9 @@ time(::Nothing) = nothing
 
 fetch_output(output, model) = output(model)
```

Comment:
What is this line here for? Is this a fallback of sorts? Do I need to replace it with `fetch_output(output, model, mask_value) = output(model)`?

Reply:
@glwagner, @simone-silvestri ?

Reply:
Yes it's a fallback, so that users can specify output which is a callable object of `model`.

Reply:
Technically we should be able to mask that output if `output(model)` returns a `Field` though.

Reply:
could change it to

```julia
function fetch_output(output, model, mask_value)
    fetched = output(model)
    if fetched isa Field
        !isnothing(mask_value) && mask_immersed(fetched, mask_value)
        return parent(fetched)
    end
    return fetched
end
```

Slightly friendlier behavior i think